src/app/pages/server-details/server-details.component.ts

```diff
@@ -70,19 +58,68 @@ export class ServerDetailsComponent implements OnInit {
   datacenterDropdown: any;
   datacenterFilters: any[] = [];
 
-  chartOptions: ChartOptions | any;
-  chartOptions2: ChartOptions | any;
-  chartOptions3: ChartOptions | any;
-
-  @ViewChild('chart1') chart!: ChartComponent;
-  @ViewChild('chart2') chart2!: ChartComponent;
-  @ViewChild('chart3') chart3!: ChartComponent;
-
+  @ViewChild('chart1') chart!: BaseChartDirective<'bar'> | undefined;
+  @ViewChild('chart2') chart2!: BaseChartDirective<'bar'> | undefined;
+  @ViewChild('chart3') chart3!: BaseChartDirective<'bar'> | undefined;
+
+  public barChartOptions: ChartConfiguration<'bar'>['options'] = {
+    // We use these empty structures as placeholders for dynamic theming.
+    scales: {
+      x: {
+        ticks: {
+          color: '#FFF',
+        },
+        grid: {
+          color: '#4B5563',
+        },
+      },
+      y: {
+        ticks: {
+          color: '#FFF',
+        },
+        grid: {
+          color: '#4B5563',
```

Comment:
let's drop the vertical grid

![image](https://github.com/SpareCores/sc-www/assets/495736/d678601b-1e19-4fa9-83b6-9869a3b925be)